vagrant/src/main/vagrant/files/install_brooklyn.sh

```diff
@@ -86,7 +86,7 @@ cp /vagrant/files/brooklyn.properties /home/vagrant/.brooklyn/
 chmod 600 /home/vagrant/.brooklyn/brooklyn.properties
 
 echo "Installing JRE"
-sudo sh -c 'export DEBIAN_FRONTEND=noninteractive; apt-get install --yes openjdk-8-jre-headless'
+sudo sh -c 'yum install -y java-1.8.0-openjdk-headless'
```

Comment:
The correct options order is `yum -y install` but this apparently works as well.